### PR TITLE
USWDS-Site - Changelog: Add entry for remove high-contrast disabled border from %block-input-styles [USWDS#5397]

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Add change log item for https://github.com/uswds/uswds/pull/5397

## Preview link

[Form change log](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-hc-input-border-changelog/components/form/#latest-updates) →

### CODE REVIEW

File: _data/changelogs/component-form.yml [hunk 1]
```diff
@@ -5,6 +5,14 @@ title: Form
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed high-contrast disabled border from text input standard variants.
+    summaryAdditional: Now the disabled border color only shows when the element is correctly disabled.
```
Comment (amyleadem) [inline _data/changelogs/component-form.yml:10]:
```suggestion
    summaryAdditional: Now the disabled border color only shows when the element is disabled.
```

File: _data/changelogs/component-form.yml [hunk 2]
```diff
@@ -5,6 +5,14 @@ title: Form
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed high-contrast disabled border from text input standard variants.
```
Comment (amyleadem) [inline _data/changelogs/component-form.yml:9]: Last release, we changed all "high-contrast" references to "forced colors". Added a couple suggestions for structure and to suggest that the current disabled styles were not intentional: 
```suggestion
    summary: Fixed a bug that caused standard text input variants to show disabled styles when in forced colors mode.
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
Added a couple of suggestions! Let me know if you have questions.

File: _data/changelogs/component-form.yml [hunk 3]
```diff
@@ -5,6 +5,14 @@ title: Form
...
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused standard text input variants to show disabled styles when in forced colors mode.
+    summaryAdditional: Now the disabled border color only shows when the element is disabled.
+    affectsStyles: true
+    affectsAccessibility: true
+    githubPr: 5397
+    githubRepo: uswds
+    versionUswds: 3.6.0
```
Comment (amyleadem) [inline _data/changelogs/component-form.yml:15]:
```suggestion
    versionUswds: 3.5.1
```

**Review by amyleadem — CHANGES_REQUESTED** [review]
One more update after our team discussion yesterday. Can you update the version to 3.5.1 and update the base branch to `release-3.5.1`?

**Comment by mahoneycm** [comment]
@amyleadem great catch! Committed your suggestion 👍

File: _data/changelogs/component-form.yml [hunk 4]
```diff
@@ -5,6 +5,14 @@ title: Form
...
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused standard text input variants to show disabled styles when in forced colors mode.
+    summaryAdditional: Now the disabled border color only shows when the element is disabled.
+    affectsStyles: true
+    affectsAccessibility: true
+    githubPr: 5397
+    githubRepo: uswds
+    versionUswds: 3.5.1
```
Comment (thisisdano) [inline _data/changelogs/component-form.yml:15]:
```suggestion
    versionUswds: N.N.N
```

While this is almost certainly going to be 3.5.1, I used dummy text here for consistency

**Review by thisisdano — APPROVED** [review]

**Review by mejiaj — APPROVED** [review]

**Review by thisisdano — APPROVED** [review]

**Pull request merged**